services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int, cinema_hall_id: int) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
+        movie=Movie.objects.get(id=movie_id)
```

Comment:
As checklist says:
Don't load the DB when it's not needed.

Before creating a movie session, an additional database query is performed to retrieve the CinemaHall and Movie objects by their id. This is an additional load on the database because you are loading objects twice when you don't need to.
Django allows you to use `movie_id` and `cinema_hall_id` directly when creating an object using a ForeignKey. In this case, you don't have to load extra objects from the database, which makes this approach more efficient.

Reply:
if you watched videos, you can remember, that use direct access to ForeginKey is a bad practice, and use .get() is a good practice